examples/plot_record_extracellular_potentials.py

```diff
@@ -81,22 +81,13 @@
 trial_idx = 0
 window_len = 10  # ms
 decimate = [5, 4]  # from 40k to 8k to 2k
-fig, axs = plt.subplots(3, 1, sharex=True, figsize=(8, 10),
-                        gridspec_kw={'height_ratios': [1, 3, 1]})
-
-# First plot the external drive statistics driving the network
-net.cell_response.plot_spikes_hist(ax=axs[0], show=False)
-axs[0].set_title('External drive spike event histograms')
```

Comment:
I ditched this plot to make it fit on the screen of a macbook air. I think it's worth it as many people may try on their laptop screens for the first time. Also, moved the dipole plot to a separate subplot for better legibility

Reply:
The inches can just be adjusted, and the plot scaled, no? Anyway, I'm fine with the change.